### ISSUE TITLE
Review Vet Center national content and workbook text for plain language changes and VA.gov style standards

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#43043]
## Issue Description

Now that the Sitewide Content team has taken over the Vet Center work, we've noticed content in the national descriptions as well as the workbook that should be rewritten in plain language and to match VA.gov's voice and style.

Before we start working with District 3 editors, we need to go through the national content to make revisions as well as the Vet Center workbook. 

Randi and Megan will make notes and edits here and then update the national content in the CMS and the workbook in Sharepoint in a new ticket. 

## Tasks
- [x] Review and edit national content for plain language and VA.gov style (Randi)
- [x] Review and edit workbook for plain language and VA.gov style (Megan)

## Acceptance Criteria
- [x] National content and workbook have been reviewed for plain language and VA.gov style guide standards

**Comment by RLHecht** [comment]
Update: I went through the centralized content and changed apostrophes throughout to curly and I made works contractions. 

Spoke to Dave Conlon, who said that we shouldn't make any substantive edits to either the Drupal content or the workbook before showing him because this is national content that may go across other facilities. 

I'll add a word doc to this with any edits we need to make to the Drupal national content and once approved we can update the nodes.

**Comment by RLHecht** [comment]
@megzehn Here are my suggested changes for the national content. Can you take a look and let me know what you think?[https://app.zenhub.com/files/133843125/d8cc0030-b9d8-4d06-9385-b9e617081045/download](https://app.zenhub.com/files/133843125/d8cc0030-b9d8-4d06-9385-b9e617081045/download)

**Comment by megzehn** [comment]
@RLHecht - these are great! Thank you so much for doing this. I added a couple more minor ones (but I want to double check them with you or Beth first[https://app.zenhub.com/files/133843125/11e2c259-3772-4687-9ecc-82e76a99b79b/download](https://app.zenhub.com/files/133843125/11e2c259-3772-4687-9ecc-82e76a99b79b/download)).

**Comment by RLHecht** [comment]
@megzehn I added some comments and other edits too. As we discussed, we may want to explain our best practice for links so that they're accessible (don't embed them, use a target and purpose). I also made a few comments about either changing the screen shots to cleaner examples or fixing the pages these came from and then posting a new screenshot. I think before we do that, I need to confirm with Dave or Jessica if we're allowed to go into the Vet Center pages once they're published to make corrections like this (not sure who "owns" the page once it's published).

[https://app.zenhub.com/files/133843125/53518627-144b-494f-9ab1-6485b688a25d/download](https://app.zenhub.com/files/133843125/53518627-144b-494f-9ab1-6485b688a25d/download)

**Comment by megzehn** [comment]
@RLHecht here's an updated version of the Workbook with comments and suggestions for paring down. There are still many examples that I need to switch, but I'm going to wait to hear from you on whether or not we're allowed to go in and make corrections on the existing examples. 

Also, I think before that, we should get Tiffany's input on what she wants to change, like you mentioned. 

[https://app.zenhub.com/files/133843125/30be7fad-edfa-4e52-b5d2-3325c2d36d75/download](https://app.zenhub.com/files/133843125/30be7fad-edfa-4e52-b5d2-3325c2d36d75/download)

**Comment by megzehn** [comment]
Hi @RLHecht, this version has the Community Engagement example included. 
[53518627-144b-494f-9ab1-6485b688a25d_Edits7.7.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/9066623/53518627-144b-494f-9ab1-6485b688a25d_Edits7.7.docx)

**Comment by RLHecht** [comment]
Moving this to validate and will create as we're waiting for feedback before implementing any of the changes. Will create a new ticket for that work.

**Comment by RLHecht** [comment]
Closing this ticket since there's a separate one to implement the changes to the workbook and national content.
